includes/html/dev-groups-overview-data.inc.php

```diff
@@ -11,7 +11,7 @@
           <div class='panel panel-default panel-condensed device-overview'>
             <div class='panel-heading'>";
 
-    echo '<i class="fa fa-th fa-fw fa-lg" aria-hidden="true"></i> <strong>Groups</strong>';
+    echo '<i class="fa fa-th fa-fw fa-lg" aria-hidden="true"></i> <strong>Included in groups</strong>';
```

Comment:
"Device Group"  not "Included in groups"  please
or "In Device Groups"

Reply:
thought a little bit about it ... maybe "Device Group Membership" is even better?
what do you think?
Also i'm thinking about title as link to Device Group Management, so it can easily reached for changing rules

Reply:
I agree, I’ll do it now